template/test-report.hbs

```diff
@@ -47,6 +54,21 @@
 
   <div class="layout-stacked__content">
     <div class="layout-container--short" style="padding-top: 35px;">   
+        <div class="filter-box">
+          <b>filter</b>
+          <hr/>
```

Comment:
Hmm, also - this top list isn't really a list of filters, but rather a list of sections to display. Perhaps they should be labelled independently to the severity filters below.

Reply:
The original thinking was filtering of the output as a whole, not precious about terminology though. I'm open to alternativr terms